docker/docker_secrets.yml

```diff
@@ -4,10 +4,22 @@
     cmd: '{{ansible_env.HOME}}/.nvm/versions/node/v24.3.0/bin/node {{ansible_env.HOME}}/.nvm/versions/node/v24.3.0/lib/node_modules/@bitwarden/cli/build/bw.js --session `cat .bw_session` get password "Fabricant Postgres Admin" > {{ansible_env.HOME}}/fabricant-prod/.secrets/postgres_admin_password.txt'
     chdir: '{{ansible_env.HOME}}/fabricant-prod'
     creates: '{{ansible_env.HOME}}/fabricant-prod/.secrets/postgres_admin_password.txt'
+- name: Deploy authentik pg admin password
+  ansible.builtin.shell:
+    cmd: '{{ansible_env.HOME}}/.nvm/versions/node/v24.3.0/bin/node {{ansible_env.HOME}}/.nvm/versions/node/v24.3.0/lib/node_modules/@bitwarden/cli/build/bw.js --session `cat .bw_session` get password "authentik postgres admin" > {{ansible_env.HOME}}/fabricant-prod/.secrets/authentik_postgres_admin_password.txt'
```

Comment:
Unrelated perhaps to this pr, but we probably should have a step in Postgres that checks for node security updates, since node packages are susceptible to supply chain attacks